bundles/org.openhab.ui/web/src/pages/settings/movable-popup-mixin.js

```diff
@@ -0,0 +1,105 @@
+import { assign, pick } from 'lodash'
+
+export default {
+  data () {
+    return {
+      movablePopup: {
+        movable: false,
+        origin: null,
+        startPos: null,
+        originalStyle: null,
+        originalCursor: null,
+        popupRef: null,
+        draggableRef: null,
+        resizeObserver: null
+      }
+    }
+  },
+  methods: {
+    // Make the given popupRef movable by dragging the draggableRef element
+    // If draggableRef is not provided, the popupRef will be used as the draggable element
+    initializeMovablePopup (popupRef, draggableRef = null) {
```

Comment:
Can you please add JSDoc here to explain the usage of this mixin?

Reply:
I'm not familiar with jsdoc... but I gave it a go.

Reply:
Thanks! I think we need to insert proper <code> fences and {@link} but I'm not familiar with it and have never worked with jsdoc before. Is there a mechanism to render and preview it?